spec/game_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../src/game'
+require_relative '../src/author'
+
```

Comment:
- [ ] Kindly fix this test case

![image](https://github.com/lordksix/catalog-things-ruby/assets/93630700/8fa7b687-ec4e-4f8a-a9c9-6a5085b92678)

Reply:
Thank you for the feedback. The test has been fixed.